django_webapps/scripts/db_population/parsers/teacher_parsers.py

```diff
@@ -0,0 +1,144 @@
+import re
+
+from email_validator import validate_email
+
+from api.models.choices.status.project import TeacherProjectStatus
+from api.models.choices.status.situational import TeacherSituationalStatus
+
+
+def parse_project_status(status_str: str) -> TeacherProjectStatus | None:
+    status_str = status_str.lower()
+    if status_str in ["идут занятия"]:
+        return TeacherProjectStatus.WORKING
+    if status_str in ["перерыв в преподавании"]:
+        return TeacherProjectStatus.FINISHED_STAYS
+    if status_str in ["закончил участие в проекте"]:
+        return TeacherProjectStatus.FINISHED_LEFT
+    if status_str in ["готов начать позже (см.коммент)"]:
+        pass
+    if status_str in ["Substitute Teacher"]:
+        pass
+    if status_str in ["Speaking Clubs"]:
+        pass
+    if status_str in ["жду ответа"]:
+        pass
+    if status_str in ["не сможет участвовать в проекте"]:
+        pass
+    if status_str in ["ищу учеников"]:
+        pass
+    if status_str in ["CV and Interview"]:
+        pass
+    if status_str in ["дубль"]:  # duplicate?
+        pass
+    return None
+
+
+def parse_situational_status(status_str: str) -> TeacherSituationalStatus | None:
+    status_str = status_str.lower()
+    if status_str in ["не вышел на связь"]:
+        return TeacherSituationalStatus.NO_RESPONSE
```

Comment:
if TID is lower than 1300, this teacher shouldn't be transferred to the new DB ([source](https://t.me/c/1808422267/2599))

Reply:
Again, only low-level parsers here